CONTRIBUTING.md

```diff
@@ -1,25 +1,37 @@
-# Contribution guide
+# Contributing to Simple Icons Font
 
-## 1. Versioning
-This package is being versioned analog to the `simple-icons` package.
-This ensures that installing a version of this package also uses the icons of the `simple-icons` package with the same
-version.
+Simple Icons welcomes contributions and corrections. Before contributing, please make sure you have read the guidelines below. If you decide to contribute anything, please do the following:
 
-Please keep in mind to update the version of this package using the `npm version` command **after** a new version of
-`simple-icons` is being released.
+1. Fork this repository
+1. Create [a new branch][github flow] from the latest `develop`
+1. Start hacking on the new branch
+1. Commit and push to the new branch
+1. Make a pull request
 
+## Local Development
 
-## 2. Building
+### Building Font Locally
 
-1. Building the font
-	```bash
-	npm run build
-	```
+* Make sure you have [NodeJS] installed. At least version 12.0.0 is required.
+* Install the dependencies using `$ npm install`.
+* Build the font using `$ npm run build`.
+* The font files can be found in the `font/` directory.
 
-1. Building the test page
-	```bash
-	npm run build:testpage
-	```
-	
-**NOTE:** Using `npm version` or `npm publish` will call the `prepublish` hook which will also build the project
-automatically.
+### Testing Font Locally
+
+* Make sure you have [NodeJS] installed. At least version 12.0.0 is required.
```

Comment:
This version is checked following the supported functionalities of the scripts or is defined using the CI as reference? Anyway is fine, just to make sure if can be improved.

Reply:
I defined it based on the CI and #62, I haven't tested if 12.0.0 actually works (only 12.16.0) do you think this should be tested?

Reply:
We can test this manually and update the version numbers if needed, but this should not block this pull request to be merged.

Reply:
I tested this manually with Node v12.0.0 (npm v6.9.0) and everything works just fine :+1: Given #62 I think we should keep v12 as a minimum.